Extractor/Types/UADataPoint.cs

```diff
@@ -88,10 +126,11 @@ public UADataPoint(UADataPoint other, double replacement)
         /// <returns>Array of bytes</returns>
         public byte[] ToStorableBytes()
         {
-            var bytes = new List<byte>(16);
+            var bytes = new List<byte>(20);
             bytes.AddRange(CogniteUtils.StringToStorable(Id));
             bytes.AddRange(BitConverter.GetBytes(Timestamp.ToBinary()));
             bytes.AddRange(BitConverter.GetBytes(IsString));
+            bytes.AddRange(BitConverter.GetBytes(Status.Code));
```

Comment:
Would be a good idea to mention in the changelog that this version of the extractor will discard locally stored datapoints that haven't yet been ingested to CDF.